File: Iniciante/1828/1828.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main() {
	int T;
	scanf("%d",&T);
	
	int i;
	for(i=1; i<=T; i++){
		char *sheldon = (char*) malloc(sizeof(char));
		char *raj = (char*) malloc(sizeof(char));
		
		scanf("%s",sheldon);
		scanf("%s",raj);
		
		if(strcmp(sheldon, raj) == 0)
			printf("Caso #%d: De novo!\n",i);
			
		else{
			if(strcmp(sheldon, "pedra") == 0){
				if( (strcmp(raj, "tesoura") == 0) || (strcmp(raj, "lagarto") == 0) )
					printf("Caso #%d: Bazinga!\n",i);
				else
					printf("Caso #%d: Raj trapaceou!\n",i);
			}
			
			if(strcmp(sheldon, "tesoura") == 0){
				if( (strcmp(raj, "papel") == 0) || (strcmp(raj, "lagarto") == 0))
					printf("Caso #%d: Bazinga!\n",i);
				else
					printf("Caso #%d: Raj trapaceou!\n",i);
			}
			
			if(strcmp(sheldon, "lagarto") == 0){
				if( (strcmp(raj, "papel") == 0) || (strcmp(raj, "Spock") == 0))
					printf("Caso #%d: Bazinga!\n",i);
				else
					printf("Caso #%d: Raj trapaceou!\n",i);
			}
			
			if(strcmp(sheldon, "Spock") == 0){
				if( (strcmp(raj, "tesoura") == 0) || (strcmp(raj, "pedra") == 0))
					printf("Caso #%d: Bazinga!\n",i);
				else
					printf("Caso #%d: Raj trapaceou!\n",i);
			}
			
			if(strcmp(sheldon, "papel") == 0){
				if( (strcmp(raj, "pedra") == 0) || (strcmp(raj, "Spock") == 0))
					printf("Caso #%d: Bazinga!\n",i);
				else
					printf("Caso #%d: Raj trapaceou!\n",i);
			}
			
		}

	}
	
	
	return 0;
}

File: Python/Iniciante/1828.py
# Problema 1828 - Beecrowd - Iniciante - Nível 4

# Definindo a quantidade de casos de teste
T = int(input())

for caso in range(T):

    # Recebendo as escolhas realizadas pelos jogadores
    escolhas = [x for x in input().split(" ")]

    # Definindo todas as combinações possíveis de vitória para Sheldon
    vitoria = [
        ["tesoura", "papel"],
        ["papel", "pedra"],
        ["pedra", "lagarto"],
        ["lagarto", "Spock"],
        ["Spock", "tesoura"],
        ["tesoura", "lagarto"],
        ["lagarto", "papel"],
        ["papel", "Spock"],
        ["Spock", "pedra"],
        ["pedra", "tesoura"]
    ]

    # Definindo condição de empate
    empate = escolhas[0] == escolhas[1]

    # Imprimindo as reações de Sheldon para todos os resultados possíveis do jogo
    if escolhas in vitoria:
        print(f"Caso #{caso+1}: Bazinga!")
    elif empate:
        print(f"Caso #{caso+1}: De novo!")
    else:
        print(f"Caso #{caso+1}: Raj trapaceou!")
